src/ia-book-actions.js

```diff
@@ -541,17 +546,30 @@ export default class IABookActions extends LitElement {
     const timerCountdown = this._shadowRoot.querySelector('timer-countdown');
     if (!timerCountdown) return;
 
-    const secondsLeft = Number(this.lendingStatus.secondsLeftOnLoan);
-    const strokeDashOffset =
-      (secondsLeft / this.loanRenewTimeConfig.loanTotalTime) * 315;
+    const secondsLeft = Number.parseInt(
+      Number(this.lendingStatus.secondsLeftOnLoan),
+      10
+    );
+    const firstStrokeDashOffset = Number.parseInt(
+      (secondsLeft / this.loanRenewTimeConfig.loanTotalTime) * 315 * 1.5,
```

Comment:
i "slowed down" the timer because its 1:1 timing made it look like it was empty way before loan ended.
this fixes
```
-> 1x - timer graphic "broke" - ~10min at end, looks empty
```